tests/components/grid.test.ts

```diff
@@ -639,6 +640,56 @@ describe("Grid component", () => {
     });
   });
 
+  describe("filter icon vertical alignment", () => {
+    let centeringOffset: number;
+    let icons: NodeList;
+    beforeEach(async () => {
+      resizeRows(model, [1], DEFAULT_CELL_HEIGHT * 2);
+      createFilter(model, "B2");
+      await nextTick();
+      icons = fixture.querySelectorAll(".o-filter-icon");
```

Comment:
```suggestion
    let icons: HTMLElement[];
    beforeEach(async () => {
      resizeRows(model, [1], DEFAULT_CELL_HEIGHT * 2);
      createFilter(model, "B2");
      await nextTick();
      icons = Array.from(fixture.querySelectorAll<HTMLElement>(".o-filter-icon"))
```
So you can  avoid having to cast `as HTMLElement` in every test. Usually it's better to do slightly more work in the beforeEach, so the tests are easier to write/read